openrouteservice/src/main/java/org/heigit/ors/api/requests/routing/RouteRequestHandler.java

```diff
@@ -167,8 +169,14 @@ public  RoutingRequest convertRouteRequest(RouteRequest request) throws StatusCo
                 throw new IncompatibleParameterException(RoutingErrorCodes.INCOMPATIBLE_PARAMETERS, RouteRequest.PARAM_ALTERNATIVE_ROUTES, "(number of waypoints > 2)");
             }
             RouteRequestAlternativeRoutes alternativeRoutes = request.getAlternativeRoutes();
-            if (alternativeRoutes.hasTargetCount())
+            if (alternativeRoutes.hasTargetCount()) {
                 params.setAlternativeRoutesCount(alternativeRoutes.getTargetCount());
+                String paramMaxAlternativeRoutesCount = AppConfig.getGlobal().getServiceParameter("routing", "profiles.default_params.maximum_alternative_routes");
```

Comment:
What happens if the default is set in the app.config, but then in the individual profile settings it is set to a large value?

Reply:
only the distance can be set per profile currently, since the check would have to be at a much later point to have profile params available, and it didn't make much sense to me to have the max routes limit set per profile anyway. want me to fix?

Reply:
never mind, it wasn't complicated to fix, it works now.